regression/cbmc/field-sensitivity-trace-counterexample/test.desc

```diff
@@ -0,0 +1,13 @@
+KNOWNBUG
+test.c
+--trace
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+x={ \.x=-?[123456789]\d* }
+--
+^warning: ignoring
+--
+This tests for the counterexample shown in the trace having sensible values.
+In this case, we are asserting that x.x != 0, so x.x should have a value other
+than 0 if the assertion fails.
```

Comment:
Since we're asserting that `x.x != 0`, in a counterexample `x.x` would be `0.`

Reply:
Indeed, this test is wrong.